### PR TITLE
Use `message.reply` instance of pinging the user

### PR DESCRIPTION
**Pull request by Syjalo**
**Please describe the changes this PR makes and why it should be merged**
To make the message look more beautiful.

**In this PR:**

- Code changes were not tested

### CODE REVIEW

File: src/listeners/messageCreate.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ client.on("messageCreate", async message => {
                             format: "`crowdin.com/translate/.../.../en-en#`"
                         })}`
                     )
-                await message.channel.send({ content: `${message.author}`, embeds: [embed] })
+                await message.reply({ embeds: [embed], allowedMentions: { repliedUser: true } })
```
Comment (ImRodry) [inline src/listeners/messageCreate.ts:92]:
```suggestion
                await message.reply({ embeds: [embed] })
```

File: src/listeners/messageCreate.ts [hunk 2]
```diff
@@ -98,7 +98,7 @@ client.on("messageCreate", async message => {
                     .setAuthor(getGlobalString("errors.wrongLink"))
                     .setTitle(getGlobalString("linkCorrectionDesc", { format: "`crowdin.com/translate/hypixel/.../en-en#`" }))
                     .setDescription(`**${getGlobalString("correctLink")}**\n${langFix.match(stringURLRegex)![0]}`)
-                await message.channel.send({ content: `${message.author}`, embeds: [embed] })
+                await message.reply({ embeds: [embed], allowedMentions: { repliedUser: true } })
```
Comment (ImRodry) [inline src/listeners/messageCreate.ts:101]:
```suggestion
                await message.reply({ embeds: [embed] })
```

**Review by ImRodry — CHANGES_REQUESTED** [review]
Please remove this and remove `repliedUser: false` from index as that's not doing anything atm

**Comment by Syjalo** [comment]
> Please remove this and remove `repliedUser: false` from index as that's not doing anything atm

Btw, you have access to my Syjalo:patch-5 repository.

**Comment by ImRodry** [comment]
I know I do but I wanted you to do it. 
Also GitHub lets you add suggestions to a batch so you can commit them all at once in the files tab

**Pull request merged**